setup.cfg

```diff
@@ -6,9 +6,24 @@ long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/nums-project/nums
 classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: C
+    Programming Language :: Python
     Programming Language :: Python :: 3
-    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development
+    Topic :: Scientific/Engineering
+    Typing :: Typed
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
     Operating System :: Unix
+    Operating System :: MacOS
```

Comment:
@elibol I'm assuming all of these operating systems are supported. Could you confirm?

Reply:
Yeah. And that's fine, I can update the operating systems to just POSIX and UNIX